docs/user/dashboard/make-dashboards-interactive.asciidoc

```diff
@@ -313,12 +313,49 @@ https://github.com/elastic/kibana/issues?q=is:issue+is:open+{{event.value}}
 . On the donut chart panel, click any chart slice, then select *Show on Github*.
 +
 [role="screenshot"]
-image:images/url_drilldown_popup.png[URL drilldown popup]
+image:images/dashboard_urlDrilldownPopup_8.3.png[URL drilldown popup]
 
 . In the list of {kib} repository issues, verify that the slice value appears.
 +
 [role="screenshot"]
-image:images/url_drilldown_github.png[Github]
+image:images/dashboard_urlDrilldownGithub_8.3.png[Open ios issues in the elastic/kibana repository on Github]
+
+[float]
+[[discover-drilldowns]]
+==== Create Discover drilldowns
+
+Discover drilldowns enable you to open *Discover* from a *Lens* dashboard panel, taking the time range, filters, and other parameters with you so the context remains the same.
+
+For example, when you want to further explore the data in a *Lens* visualization, you can create a drilldown that opens the visualization data in *Discover*.
+
+[role="screenshot"]
+image:images/dashboard_discoverDrilldown_8.3.gif[Drilldown on bar vertical stacked chart that navigates to Discover]
+
+Discover drilldowns are supported only by *Lens* panels. 
+
+[float]
+===== Create the Discover drilldown
+
+Create a drilldown that opens *Discover* from the <<gs-get-data-into-kibana,*Sample web logs*>> data *[Logs] Web Traffic* dashboard.
+
+. Click *Edit*, open the panel menu for the *[Logs] Bytes distribution* bar vertical stacked chart, then select *Create drilldown*.
+
+. Click *Open in Discover*.
+
+. Give the drilldown a name. For example, `View bytes distribution in Discover`.
+
+. To take the time range, filters, and other parameters with you, select *Apply filter*.
+
+. To open the Discover drilldown in a new tab, select *Open in new tab*.
+
+. Click *Create drilldown*.
+
+. Save the dashboard.
+
+. On the *[Logs] Bytes distribution* bar vertical stacked chart, click a bar, then select *View bytes distribution in Discover*.
```

Comment:
So, if the user is doing this, it will go to Discover filtering down to the clicked bar. They can also go to Discover without the filter by using the panel context menu action ("Explore data in Discover"). It feels like we should mention this here.